Tasks/google-play-release/GooglePlay.ts

```diff
@@ -107,13 +105,20 @@ async function run() {
 
         let requireTrackUpdate = false;
         if (shouldUploadApks) {
-            tl.debug(`Uploading ${apkFileList.length} APK(s).`);
+            tl.debug(`Uploading ${Object.keys(apkToObbMapping).length} APK(s).`);
             requireTrackUpdate = true;
 
-            for (const apkFile of apkFileList) {
+            for (const apkFile of Object.keys(apkToObbMapping)) {
                 tl.debug(`Uploading APK ${apkFile}`);
+                console.log(apkFile);
```

Comment:
This line isn't needed.  `tl.debug` immediately above is already providing what this gives.  If you need to see what files are being uploaded, then set your `system.debug=true` when doing a build.